frontend/src/app/main/ui/workspace/shapes/common.cljs

```diff
@@ -36,8 +37,9 @@
         nil
 
         (= type :frame)
-        (do (dom/stop-propagation event)
-            (st/emit! (dw/start-move-selected)))
+        (when selected?
+          (do (dom/stop-propagation event)
+              (st/emit! (dw/start-move-selected))))
```

Comment:
Now it works to unselect shape by clicking inside the frame. But then it brokes to select the frame with double click :P